app/stack/new-database/route.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  requireAuthentication: true,
```

Comment:
If you prefer that authentication always be explicit, but don't want to use a base class, you could assert that the `requireAuthentication` property is not undefined. That seems ok.

As they stand, I would be confused about what the default behavior is from looking at usage.